tests/test_nusantara.py

```diff
@@ -434,7 +433,7 @@ def test_entities_offsets(self, dataset_nusantara: DatasetDict):
                 for example in dataset_nusantara[split]:
 
                     example_id = example["id"]
-                    example_text = _get_example_text(example)
+                    example_text = _get_example_text(example, "entities")
```

Comment:
Also revert this one, since we will remove the `subcomponent` parameter in this case